### ISSUE TITLE
Can't handle the HTTP exception on async API

### ISSUE THREAD

**Issue by yoshiya0503** [line/line-bot-sdk-python#363]
- Make sure your issue is **related to** the LINE Bot SDK. For general questions about LINE bots, please ask in https://www.linebiz.com/contact/
- Provide detailed information about the issue you had with the SDK as below

## System Informations

* Python version: 3.9.0
* SDK version: 2.0.1
* OS: mac OS 11.6

## Expected Behavior
handle exceptions as expected

## Current Behavior
all http exceptions will no handled, but occured
'coroutine' object has no attribute 'items'

## Steps to Reproduce

1. create some example like this https://github.com/line/line-bot-sdk-python/pull/342/files 
2. insert import asyncio to the exmample
3. insert asyncio.sleep(40) out here https://github.com/line/line-bot-sdk-python/pull/342/files#diff-6cb84c37fb2f0d772b048005d97a7f10fc4f3342bb7bacccdee2a3b40fff56c1R69
4. fire webhooks
5. the exception should raise like this

**LineBotApiError: status_code=400, request_id=b87fc297-873c-4520-a298-eb057a576701, error_response={"details": [], "message": "Invalid reply token"}, headers={'Server': 'openresty', 'Date': 'Sat, 27 Nov 2021 13:56:31 GMT', 'Content-Type': 'application/json', 'Content-Length': '33', 'Connection': 'keep-alive', 'Cache-Control': 'no-cache, no-store, max-age=0, must-revalidate', 'Expires': '0', 'Pragma': 'no-cache', 'x-content-type-options': 'nosniff', 'x-frame-options': 'DENY', 'x-line-request-id': 'b87fc297-873c-4520-a298-eb057a576701', 'x-xss-protection': '1; mode=block'}**

6. but we got 'coroutine exception' at here https://github.com/line/line-bot-sdk-python/blob/master/linebot/async_api.py#L1716
7. the cause of this. response.json of aiohttp is coroutine. so we should
```
await response.json
```

Is this async code was generated?

**Comment by louis70109** [comment]
As you say, maybe we should change like the below?

```
self.__check_error( await response)
```

**Comment by yoshiya0503** [comment]
It subtle mistake.

response object of aiohttp is not awaitable. this is a simple object.
It will cause like this syntax error.

```
object AiohttpAsyncHttpResponse can't be used in 'await' expression
```

awaitable object is

```
response.json
```

**Comment by yoshiya0503** [comment]
For example. To fix this issue on 'cheap method'.

https://github.com/line/line-bot-sdk-python/pull/364/files#diff-9aca243eb8cba00a4aa078ab3c6df50cae9e165cb4c3779a6114c90f5c61b3acR1650

don't marge this PR.
this is EXMPLE for fix.

**Comment by yoshiya0503** [comment]
Oh... Unfortunately...

I found same errors not only error handling.

all json response has same errors.

```
response.json => await response.json
```

**Comment by louis70109** [comment]
Closed with  #364
